lms/djangoapps/courseware/views/views.py

```diff
@@ -510,10 +510,9 @@ def course_about(request, course_id):
         ecommerce_bulk_checkout_link = ''
         professional_mode = None
         ecomm_service = EcommerceService()
-        if ecomm_service.is_enabled(request.user) and (
-                CourseMode.PROFESSIONAL in modes or CourseMode.NO_ID_PROFESSIONAL_MODE in modes
-        ):
-            professional_mode = modes.get(CourseMode.PROFESSIONAL, '') or \
+        if ecomm_service.is_enabled(request.user) and (CourseMode.HONOR in modes
```

Comment:
Nitpick: I would break the lines in a different manner here. Operators should always be at the end of the line, long statements usually look better if you wrap them into brackets, backslashes add clutter.
```python
if (ecomm_service.is_enabled(request.user) and
        (CourseMode.HONOR in modes or CourseMode.NO_ID_PROFESSIONAL_MODE in modes)):
    professional_mode = (
        modes.get(CourseMode.HONOR, '') or
        modes.get(CourseMode.NO_ID_PROFESSIONAL_MODE, '')
    )
```